helm/install.go

```diff
@@ -355,3 +363,33 @@ func Install(helmInstall *phelm.Install, kubeConfig []byte) error {
 	log.Info("Helm install finished")
 	return nil
 }
+
+// watchTillerUntilReady waits for the tiller pod to become available.
+//
+// Returns true if it exists. If the timeout was reached and it could not find the pod, it returns false.
+func watchTillerUntilReady(namespace string, client *kubernetes.Clientset, timeout time.Duration, newImage string) bool {
+	deadlinePollingChan := time.NewTimer(timeout).C
```

Comment:
minor: we already have a library which handles backoff, https://github.com/banzaicloud/pipeline/tree/master/internal/backoff

Reply:
Removing this part as this is already handled in `WaitingForTillerComeUp`